doc/classes/EditorToaster.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="EditorToaster" inherits="HBoxContainer" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+	</brief_description>
+	<description>
+	</description>
```

Comment:
This technically needs to be filled too, but it's not really worth it to expose a class just to expose some constants. Maybe they can be part of EditorInterface too.

Reply:
Hmm, gotcha, so you think I should copy the enums into EditorInterface (and have them in EditorInterface and EditorToaster) or move the enum into EditorInterface and reference it in EditorToaster? Which do you think is better?

Reply:
Hmm, duplicating the enum is annoying 😑 So in that case it's probably fine to expose the class.

I'm wondering what is a better option: leaving it as is or moving the `push_toast` method to EditorToaster and make the toaster accessible via `get_editor_toaster()` method in EditorInterface. The latter would be more consistent with other classes, but the toaster has only a single exposed method.

With either option you'll need to fill in the descriptions (even with some very basic info).

Reply:
Agreed with duping the enum, I'll make the change to expose it as `get_editor_toaster` and we can see if we like that.